Ginger/Ginger/Reports/ExecutionResultsConfiguration.xaml.cs

```diff
@@ -232,6 +250,35 @@ private void PublishLogToCentralDBRadioButton_CheckedHandler(object sender, Rout
                 xPublishingPhase.Visibility = Visibility.Collapsed;
                 xPublishingPhasePanel.Visibility = Visibility.Collapsed;
             }
-        }               
+        }
+
+        private void SetExecutionLoggerRadioButtonToOff()
+        {
+            if (_selectedExecutionLoggerConfiguration.PublishLogToCentralDB == ExecutionLoggerConfiguration.ePublishToCentralDB.No)
+            {
+                if (xPublishLogToCentralDBRadioBtnPanel != null && xPublishLogToCentralDBRadioBtnPanel.Children != null && xPublishLogToCentralDBRadioBtnPanel.Children.Count != 0)
+                {
+                    for (int i = 0; i < xPublishLogToCentralDBRadioBtnPanel.Children.Count; i++)
```

Comment:
Instead of looping thorugh children, please change checked property directy accessing radiobutton by name

Reply:
These radio buttons are dynamically added through code and are not publicly available to access.